packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.Props.ts

```diff
@@ -201,6 +201,11 @@ export interface IDetailsListProps extends React.Props<DetailsList>, IWithViewpo
    * The default implementation will virtualize when this callback is not provided.
    */
   onShouldVirtualize?: (props: IListProps) => boolean;
+
+  /**
+   * Optional class name to add to the cell of a checkbox
+   */
+  checkboxClassName?: string;
```

Comment:
We already have on `check...` props, should we go all the way and take `ICheckProps`?

Reply:
Okay, synced offline. the DetailsCheck and CheckProps have their own issues so it might be fine.

Only complaint, while the comment explains correctly that the class is added to the cell of the checkbox, the name doesn't match that. Can we find a better one?